lib/auth/login/widgets/login_form.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flex_storefront/auth/login/cubits/login_cubit.dart';
+import 'package:flex_storefront/auth/login/cubits/login_state.dart';
+import 'package:flex_storefront/flex_ui/tokens/sizes.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+import 'package:form_builder_validators/form_builder_validators.dart';
+import 'package:icons_plus/icons_plus.dart';
+
+class LoginForm extends StatefulWidget {
+  const LoginForm({
+    super.key,
+  });
+
+  @override
+  State<LoginForm> createState() => _LoginFormState();
+}
+
+class _LoginFormState extends State<LoginForm> {
```

Comment:
From what I see, this widget has no reason to be stateful (no local variable and no lifecycle method overriden)